cmd/root.go

```diff
@@ -56,15 +59,43 @@ var RootCmd = &cobra.Command{
 			panic(err)
 		}
 
-		currTime := time.Now().Unix()
+		var commuteTime int64
 		minute := 60
 		interval := int64(interval * minute)
 		var traveTime int64 // time leaving
 		var info directions.CommuteInfo
 		var shortest int = 0
 		var optimalTime string
 		var bestHour int
+		today := true
```

Comment:
determining if the date is today should be its own helper method in the Utils package not a flag manipulated by the format date logic.
```go
utils.DateIsToday(start)
```
can be called wherever, in this case on line 127 `if i == 0 && utils.DateIsToday(start) `